routing/providerquerymanager/providerquerymanager.go

```diff
@@ -114,7 +114,7 @@ func WithMaxTimeout(timeout time.Duration) Option {
 
 // WithMaxInProcessRequests is the maximum number of requests that can be
 // processed in parallel. If this is 0, then the number is unlimited. Default
-// is defaultMaxInProcessRequests (16).
```

Comment:
Perhaps we should make those defaults public, because reading the docs and finding a default is defined in a private const and you have to go read the code for that is not fantastic.

Reply:
Should these consts be exported, or should there be a different set of exported consts for setting up the default provider query manager?

Reply:
We have prior art in https://github.com/ipfs/boxo/blob/main/bitswap/internal/defaults/defaults.go – maybe for now just add there?

Reply:
consts are exported now

Reply:
@lidel Done. Values for default bitswap client come from `bitswap/internal/defaults/defaults.go`